k8s/voucher/apptest/tester/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM radial/busyboxplus:curl
```

Comment:
It's worth to check the possibility to use a busybox image. 

I will send you an internal link in private.

Reply:
done.